README.rst

```diff
@@ -9,6 +9,16 @@ Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
 Currently, Windows PowerShell, Command Prompt, and Git Shell for
 Windows are supported with limitations (See `Windows Issues`_).
 
+This product is supported by the Cybersecurity Development team at the 
+University of Illinois, on a best-effort basis. As of the last update to 
+this README, the expected End-of-Life and End-of-Support dates of this 
+product are 05 October 2025.
```

Comment:
Let's change 05 October 2025 to October 5, 2025. I like Day/Month/Year or better Year/Month/Day in code, but in writing it seems strange.

Reply:
I am indifferent to the date format. 
I prefer this format as this seems more in line with other examples of EOL available online.

Reply:
Sounds good. I'll update the format here.